subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -627,7 +622,20 @@ static void isr_rx(void *param)
 		e->type = EVENT_DONE_EXTRA_TYPE_SYNC_ISO_TERMINATE;
 
 		lll_isr_cleanup(param);
+
 		return;
+
+	/* Check if BIG Channel Map Update */
+	} else if (lll->chm_chan_count) {
+		uint16_t event_counter;
+
+		event_counter = lll->payload_count / lll->bn;
+		if (((event_counter - lll->ctrl_instant) & 0xffff) <= 0x7fff) {
```

Comment:
Do we have a macro for `0x7FFF`? Not what exactly that value represents. 

`& 0xffff` seems redundant since `event_count` is a uint16_t, so the value can not be above 0xFFFF in any case.